modules/nf-core/gridss/gridss/tests/main.nf.test

```diff
@@ -0,0 +1,182 @@
+                input[2] = [ [id:'fasta_fai'],
+                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta.fai', checkIfExists: true)
+                ]
+                input[3] = BWA_INDEX.out.index
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match() },
+                { assert path(process.out.vcf.get(0).get(1)).linesGzip.contains("##fileformat=VCFv4.2") }
+            )
+        }
+
+    }
+
+    test("human - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                        file(params.modules_testdata_base_path + '/genomics/homo_sapiens/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true), //inputs
+                        [] //assembly
+                        ]
+                input[1] = [ [id:'fasta'],
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta', checkIfExists: true)
+                        ]
+                input[2] = [ [id:'fasta_fai'],
+                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta.fai', checkIfExists: true)
+                        ]
+                input[3] = [ [], [] ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match() },
+                { assert path(process.out.vcf.get(0).get(1)).linesGzip.contains("##fileformat=VCFv4.2") }
+            )
+        }
+
+    }
+
+    test("human - bam - assembly") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                        file(params.modules_testdata_base_path + '/genomics/homo_sapiens/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true), //inputs
+                        file(params.modules_testdata_base_path + '/genomics/homo_sapiens/illumina/bam/test2.paired_end.sorted.bam', checkIfExists: true) //assembly
+                        ]
+                input[1] = [ [id:'fasta'],
+                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta', checkIfExists: true)
+                        ]
+                input[2] = [ [id:'fasta_fai'],
+                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta.fai', checkIfExists: true)
+                        ]
+                input[3] = [ [], [] ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.assembly[0][1]).name,
```

Comment:
Is this a bam file? As there is a [nf-test extension for bam files](https://nvnieuwk.github.io/nft-bam/latest/)

Reply:
Is there another module who uses this were I can have a look?